frontend/packages/helm-plugin/src/utils/helm-utils.ts

```diff
@@ -92,26 +92,37 @@ export const getChartURL = (
   return chartData?.urls[0];
 };
 
+export const getChartRepositoryTitle = (
+  chartRepositories: K8sResourceKind[],
+  chartRepoName: string,
+) => {
+  const chartRepository = chartRepositories.find((repo) => repo.metadata.name === chartRepoName);
```

Comment:
will `chartRepositories` always be there or good to have check as it's optional in `getChartEntriesByName`. WDYT?

Reply:
Yes, in this case the utility is going to be specifically used to find correct repository and get it's title. So it can only be called when there are chartRepositories are available.